packages/styles/forms.css

```diff
@@ -364,7 +363,10 @@ textarea.Field--has-error:focus:hover,
   color: #3c7aae;
 }
 
-.Checkbox__overlay--disabled.Icon--checkbox-checked,
+.Checkbox__overlay--disabled.Icon--checkbox-checked {
+  color: #78a6d8;
+}
+
 .Checkbox__overlay--disabled.Icon--checkbox-unchecked {
   color: #ccc;
```

Comment:
While we're here, we should update this to use our global variable (similar to above):

```css
color: var(--field-checkbox-unchecked-disabled-color, var(--gray-40))
```
```